readme.md

```diff
@@ -12,33 +12,33 @@ npm install --save-dev chrome-webstore-upload
 
 ## Setup
 
-You will need a Google API `clientId`, `clientSecret` and `refreshToken`. Use [the guide]( https://github.com/fregante/chrome-webstore-upload-keys).
+You will need a Google API `clientId`, `clientSecret` and `refreshToken`. Use [the guide](https://github.com/fregante/chrome-webstore-upload-keys).
 
 ## Usage
 
-All methods return a  promise.
+All methods return a promise.
 
 ### Create a new client
 
 ```javascript
-import chromeWebstoreUpload from 'chrome-webstore-upload';
+import chromeWebstoreUpload from "chrome-webstore-upload";
```

Comment:
Please don't make unrelated changes. This was probably Prettier making changes "on save". All of these should be reverted for the PR to be merged

Reply:
It's fixed